manifests/logging/channel.pp

```diff
@@ -0,0 +1,43 @@
+# Define new channel for logging
+define dns::logging::channel (
+  Optional[Stdlib::Absolutepath] $file_path          = undef,
+  Optional[String] $file_size                        = undef,
+  Optional[Integer] $file_versions                   = undef,
+  Enum['file', 'null', 'stderr', 'syslog'] $log_type = undef,
+  Integer $order                                     = 10,
+  Optional[Enum['no', 'yes']] $print_category        = undef,
+  Optional[Enum['no', 'yes']] $print_severity        = undef,
+  Optional[Enum['no', 'yes']] $print_time            = undef,
+  Optional[String] $severity                         = undef,
+  Optional[String] $syslog_facility                  = undef,
+) {
+  $channel_name = $title
+
+  if ($order < 2) or ($order > 98) {
```

Comment:
It'd recommend using `Integer[2, 98]` for `$order` so you don't need to write this check.